devopstools-jenkins-chefhabitat-kubernetes/azuredeploy.json

```diff
@@ -0,0 +1,417 @@
+{
+    "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "adminUsername": {
+            "type": "string",
+            "metadata": {
+                "description": "user name for the virtual machines."
+            }
+        },
+        "azureUsername": {
+            "type": "string",
+            "metadata": {
+                "description": "azure login name"
+            }
+        },
+        "azurePassword": {
+            "type": "securestring",
+            "metadata": {
+                "description": "azure login password"
+            }
+        },
+        "authenticationType": {
+            "type": "string",
+            "defaultValue": "sshPublicKey",
+            "metadata": {
+                "description": "authenticationType for virtula machines"
+            }
+        },
+        "sshPublicKey": {
```

Comment:
this should have a defaultValue since authn type is a parameter

Reply:
removed authentication type from parameter section